ui/app/pages/first-time-flow/seed-phrase/seed-phrase-intro/index.scss

```diff
@@ -0,0 +1,77 @@
+.seed-phrase-intro {
+  &__sections {
+    display: flex;
+
+    @media screen and (min-width: $break-large) {
+      flex-direction: row;
+    }
+
+    @media screen and (max-width: 830px) {
+      flex-direction: column;
+    }
+  }
+
+  &__left {
+    flex: 3;
+    min-width: 0;
+  }
+
+  &__right {
+    flex: 1;
+    min-width: 0;
+
+    @media screen and (max-width: 830px) {
+      margin-top: 24px;
+    }
+  }
+
+  &__title {
```

Comment:
This class, for example, can be expressed with Typography
```jsx
<Typography color={COLORS.BLACK} variant={TYPOGRAPHY.H1} boxProps={{ marginTop: 0, marginBottom: 4 }} >
```

Reply:
Done, and this rule removed as a result